### DISCUSSION TITLE
UE OTA and other log packet contents

### DISCUSSION THREAD

**Discussion by pujashuk** [aligungr/UERANSIM#431] [Q&A]
Hi, 

First of all, thanks a lot for this gigantic task of creating this UERANSIM! I have a query regarding the checking of the logs on UERANSIM generated. If I want to see the signalling content like RRC Reconfiguration and System Information Types, to get the hang of what is really happening and the ARFCN, frequency and BEAM ID info. 

Can you please help?

**Comment by aligungr** [comment]
Hi @pujashuk 

You can use this Wireshark plugin to monitor RRC messaging:
https://github.com/louisroyer/RLS-wireshark-dissector

**NOTE:** Most of the functionalities of RRC protocol are not implemented yet. So you won't be able to see some of the information you mentioned.

Best Regards

**Reply by pujashuk** [reply]
Hi @aligungr 

Thanks for the quick aid. I am facing some issue with the current setup. I have a virtual box installed on my PC where three instances of VMs are configured. Where should I add the wireshark plugin?
I tried installing wireshark on PC and installed the plugin on both the virtual box and the PC. However, I see no packets being received on the PC. Do you have any suggestions on this?

**Reply by aligungr** [reply]
You should install the plugin in the UE's and gNodeB's machine (i.e. UERANSIM's machine). And make sure that you listen `any` interface. And the plugin installed successfully (this may be challenging but please refer to Wireshark documentation)
